src/main/java/io/logz/guice/jersey/JerseyServer.java

```diff
@@ -28,9 +29,16 @@
 
     JerseyServer(JerseyConfiguration jerseyConfiguration,
                  Supplier<Injector> injectorSupplier) {
+        this(jerseyConfiguration, injectorSupplier, jettyServer -> {});
```

Comment:
Remove old constructor, this is internal class that should only be initialized using the guice